static/sass/styles.scss

```diff
@@ -1,151 +1,11 @@
 @charset "UTF-8";
-/* ---------------------------------------------------------------
-------------------------------------------------------------------
-	DO NOT EDIT STYLES.CSS
-	It is autogenerated.
 
-	Project:		Ubuntu Front-End
-	Author:			Web Team at Canonical Ltd
-	Last edited by:
-	Last edited on:
+$color-brand: #e95420;
+$color-accent: #2c001e;
+$color-link: $color-brand;
 
-  	CONTENTS:
-	-----------------------------------------------------------------
-	@section general
-	@section main nav
-	@section footer
-	@section UA service-description
+// Import Vanilla Framework
+@import "../../node_modules/vanilla-framework/scss/build";
```

Comment:
Your importing vanilla but not including it?

Reply:
`build.scss` imports and includes

<img width="569" alt="screen shot 2017-09-01 at 15 41 44" src="https://user-images.githubusercontent.com/505570/29974840-23ec5078-8f2c-11e7-87e8-80264102ac84.png">

Reply:
I thought we imported and included vanilla in projects and `build.scss` was only used for building vanilla from within the vanilla project?